libsolidity/analysis/DeclarationTypeChecker.cpp

```diff
@@ -383,6 +383,7 @@ void DeclarationTypeChecker::endVisit(VariableDeclaration const& _variable)
 					string(_variable.isReturnParameter() ? "return " : "") +
 					"parameter in" +
 					string(_variable.isExternalCallableParameter() ? " external" : "") +
+                    string(_variable.isPublicCallableParameter() ? " public" : "") +
```

Comment:
Your line is indented with spaces while the rest of the code is using tabs. This is probably why [`chk_coding_style`](https://circleci.com/gh/ethereum/solidity/860718) job is failing.